includes/elements/container.php

```diff
@@ -466,6 +466,24 @@ protected function register_container_layout_controls() {
 			'separator' => 'none',
 		];
 
+		$active_breakpoints = Plugin::$instance->breakpoints->get_active_breakpoints();
```

Comment:
@ArmanPurtoian `$active_breakpoints = Plugin::$instance->breakpoints->get_active_breakpoints();` has already been done on line 407. Can you use that variable?